src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -542,16 +542,20 @@ SHAMapStoreImp::dbPaths()
 
         journal_.error() << "state db error: " << std::endl
                 << "  writableDbExists " << writableDbExists
-                << " archiveDbExists " << archiveDbExists << std::endl
+                << " archiveDbExists " << archiveDbExists << '\n'
                 << "  writableDb '" << state.writableDb
                 << "' archiveDb '" << state.archiveDb << "'"
-                << std::endl << std::endl
```

Comment:
Doesn't this cause issues on Windows? Why mix `std::endl` and `\n` in the first place?

Reply:
I don't think it's a problem. The `\n` works correctly when printing to the console; worse case scenario, it doesn't print over multiple lines in the file and that's sorta-kinda OK. The only difference here is that `std::endl` will flush `stdout` buffers.

Reply:
@MarkusTeufelberger @nbougalis I've removed std::endl -- at the least, it's kind of ugly to mix the two types of line endings. And when rippled writes to the console, it's to stderr, which is unbuffered, so std::endl doesn't even have an effect different from '\n'.